src/sass/media.scss

```diff
@@ -0,0 +1,131 @@
+$phone: 568px;
+$tablet: 768px;
+$desktop: 1024px;
+
+@media only screen and (min-width: $phone) {
+  main {
+    .weather {
+      flex-direction: row;
+      flex-wrap: wrap;
+      justify-content: center;
+      .overcast {
+        margin-top: 70px;
+        h4 {
+          display: none;
+        }
+      }
+    }
+    .forecast {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      justify-content: center;
+      h4 {
+        flex-basis: auto;
+        width: 100%;
+        margin-bottom: 80px;
+      }
+      > div {
+        position: relative;
+        text-align: left;
+        margin-right: 20px;
+        > span {
+          position: absolute;
+          bottom: 55px;
+          left: 0;
+        }
+        > p {
+          margin: 0;
+        }
+      }
+    }
```

Comment:
Usually I would prefer to avoid `> div`
Could really impact maintainability in the future. 
But not a mistake ofc

Reply:
ok, thanks!